java-accessapproval/.OwlBot.yaml

```diff
@@ -12,24 +12,22 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-
-deep-remove-regex:
-- "/java-accessapproval/grpc-google-.*/src"
-- "/java-accessapproval/proto-google-.*/src"
-- "/java-accessapproval/google-.*/src"
-- "/java-accessapproval/samples/snippets/generated"
-
-deep-preserve-regex:
-- "/java-accessapproval/google-cloud-accessapproval/src/test/java/com/google/cloud/accessapproval/v1/it"
-
-deep-copy-regex:
-- source: "/google/cloud/accessapproval/(v.*)/.*-java/proto-google-.*/src"
-  dest: "/owl-bot-staging/java-accessapproval/$1/proto-google-cloud-accessapproval-$1/src"
-- source: "/google/cloud/accessapproval/(v.*)/.*-java/grpc-google-.*/src"
-  dest: "/owl-bot-staging/java-accessapproval/$1/grpc-google-cloud-accessapproval-$1/src"
-- source: "/google/cloud/accessapproval/(v.*)/.*-java/gapic-google-.*/src"
-  dest: "/owl-bot-staging/java-accessapproval/$1/google-cloud-accessapproval/src"
-- source: "/google/cloud/accessapproval/(v.*)/.*-java/samples/snippets/generated"
-  dest: "/owl-bot-staging/java-accessapproval/$1/samples/snippets/generated"
-
 api-name: accessapproval
+deep-copy-regex:
+  - dest: /owl-bot-staging/java-accessapproval/$1/proto-google-cloud-accessapproval-$1/src
+    source: /google/cloud/accessapproval/(v.*)/.*-java/proto-google-.*/src
+  - dest: /owl-bot-staging/java-accessapproval/$1/grpc-google-cloud-accessapproval-$1/src
+    source: /google/cloud/accessapproval/(v.*)/.*-java/grpc-google-.*/src
+  - dest: /owl-bot-staging/java-accessapproval/$1/google-cloud-accessapproval/src
+    source: /google/cloud/accessapproval/(v.*)/.*-java/gapic-google-.*/src
+  - dest: /owl-bot-staging/java-accessapproval/$1/samples/snippets/generated
+    source: /google/cloud/accessapproval/(v.*)/.*-java/samples/snippets/generated
+deep-preserve-regex:
+  - /java-accessapproval/google-cloud-accessapproval/src/test/java/com/google/cloud/accessapproval/v1/it
+  - /java-accessapproval/google-.*/src/main/resources/META-INF/native-image/*
+  - /java-accessapproval/google-.*/src/test/resources/META-INF/native-image/*
+deep-remove-regex:
+  - /java-accessapproval/grpc-google-.*/src
+  - /java-accessapproval/proto-google-.*/src
+  - /java-accessapproval/google-.*/src
```

Comment:
Can you keep the order of the sections? The old order (deep-remove-regex, deep-preserve-regex, and deep-copy-regex) matches OwlBot's steps go/owlbot-howto#githubowlbotyaml.

Reply:
I see. I can keep the order for clarity of the owl-bot steps.